frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -290,10 +294,11 @@ local footerTextGeneratorMap = {
         local prefix = symbol_prefix[symbol_type].mem_usage
         local statm = io.open("/proc/self/statm", "r")
         if statm then
-            local dummy, rss = statm:read("*number", "*number")
+            local infos = statm:read("*all")
             statm:close()
+            local rss = infos:match("^%S+ (%S+) ")
             -- we got the nb of 4Kb-pages used, that we convert to Mb
-            rss = math.floor(rss * 4096 / 1024 / 1024)
+            rss = math.floor(tonumber(rss) * 4096 / 1024 / 1024)
```

Comment:
You/we just don't want this part to appear in the diff :) it's not your work, just the glitch I'm talking about.

Reply:
That's exactly what I did - thank you!